docs/user_guide/hopsworks/python.rst

```diff
@@ -88,15 +84,44 @@ Enter the search term and select the version from the drop down.
 
     Installing library using the search
 
+**Installation option 3: Install from .whl or .egg file**
+
+Select the uploaded package to install by selecting it in the file browser.
+
+.. _python10.gif: ../../_images/python/python10.gif
+.. figure:: ../../imgs/python/python10.gif
+    :alt: Install library from file browser
+    :target: `python10.gif`_
+    :align: center
+    :figclass: align-center
+
+    Installing package uploaded in the file browser
+
+**Installation option 4: Install from .git repository**
+
+To install from a git repository simply provide the repository URL. The same you would provide to git in the command line using *pip install git+{repo_url}*.
```

Comment:
The URL you should provide is the same as you would enter on the command line using